release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,129 @@
 
-|ArgoCD backend|`@roadiehq/backstage-plugin-argo-cd-backend`|2.11.3|Continuous deployment
+=== Configuration change in the GitHub Org catalog provider  
 
-|API Documentation|`@backstage/plugin-api-docs`|0.9.12|Continuous deployment
+The GitHub Org catalog module for the new backend system switched from an obsolete implementation to a new one, that is, `GithubMultiOrgProvider`. This new implementation brings in changes to the default plugin configuration settings.
 
-|App backend|`@backstage/plugin-app-backend`|0.3.54|Continuous deployment
+=== New Scaffolder Identity client
 
-|Auth backend|`@backstage/plugin-auth-backend`|0.19.4|Security
+Identity client functioning differs slightly between the one provided by the new backend system Scaffolder plugin, and the one used in the previous release:
 
-|Auth node|`@backstage/plugin-auth-node`|0.4.0|Security
+* The previous release uses the `DefaultIdentityClient` function without any specific option
+* New backend system Scaffolder does not use the default identity service, but still uses the deprecated scaffolder-dedicated `buildDefaultIdentityClient` function, which introduces a slight functioning change in the identity client. 
 
-|Catalog frontend|`@backstage/plugin-catalog`|1.14.0|Continuous deployment
+=== Enable all GitLab actions
+{product} {product-version} enables all GitLab actions by default.
 
-|Catalog backend|`@backstage/plugin-catalog-backend`|1.14.0|Continuous deployment
+=== Events support redesign in progress for the new backend system
+The `Events` plugin is not completely supported by the new backend system, and thus its usefulness is greatly reduced, if not completely broken.
+A complete redesign of the `Events` plugin support for the new backend system is in progress, but was still not available to integrate it to this release of {product}. This limitation temporarily prevents third parties from fully using events in plugins or modules they want to provide as dynamic plugins for {product}.
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+=== RBAC Plugin
+In the new backend system, automatically getting the IDs of all dynamically installed plugins is not possible. This means that permissions of dynamic plugins supporting RBAC will not be automatically displayed in the RBAC UI.
+As a workaround, you can manually add a list of dynamic plugin IDs supporting RBAC in the {product-short} application configuration file.
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+== Support for Elastic Kubernetes Services (EKS)
 
-|Catalog React|`@backstage/plugin-catalog-react`|1.8.5|Continuous deployment
+You can now integrate your {product} application with Amazon Web Services (AWS), which optimizes your development workflows within the AWS ecosystem.
```

Comment:
Similarly as with EKS, this is not really true. The EKS support is mainly about supporting installing RHDH into EKS clusters. There is no optimizations of the development workflow (not even sure it would mean :-/ )

Reply:
For better clarity, rephrasing this part to:
You can now install and use {product} on an EKS cluster.